lib/nerves_hub_user_api/api.ex

```diff
@@ -147,8 +147,14 @@ defmodule NervesHubUserAPI.API do
     ca_cert_path
     |> File.ls!()
     |> Enum.map(&File.read!(Path.join(ca_cert_path, &1)))
-    |> Enum.map(&Certificate.from_pem!/1)
+    |> Enum.reduce([], fn bin, acc ->
+      case Certificate.from_pem(bin) do
+        {:ok, cert} -> [cert | acc]
+        {:error, _} -> acc
+      end
+    end)
     |> Enum.map(&Certificate.to_der/1)
+    |> Enum.reverse()
```

Comment:
I am not sure this is needed...

Reply:
Agreed. I think we can take this out since order doesn't really matter. Otherwise, LGTM